### PR TITLE
[7.39.x] RHPAM-3027: Using Task Assigning with DDL prevents users from completing tasks

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hi @rsynek would you mind review this PR?
Thanks.

It's related with https://github.com/kiegroup/jbpm/pull/1690 but can be merged separately

**Comment by wmedvede** [comment]
jenkins retest this

File: kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-runtime/src/main/java/org/kie/server/services/taskassigning/runtime/command/DelegateAndSaveCommand.java [hunk 1]
```diff
@@ -38,6 +38,9 @@
     static final String TASK_MODIFIED_ERROR_MSG = "Task: %s was modified by an external action since the last executed plan," +
             " current status is %s but the expected should be in %s";
 
+    static final String TASK_MODIFIED_ERROR_MSG_1 = "Task: %s was not found, it might have been deleted by an external action since the last executed plan." +
```
Comment (rsynek) [inline kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-runtime/src/main/java/org/kie/server/services/taskassigning/runtime/command/DelegateAndSaveCommand.java:41]: The same concern as in the PR for the master branch.

**Comment by sutaakar** [comment]
FYI the CI job timeout was extended so the PR check should finish now.

**Comment by wmedvede** [comment]
jenkins execute full downstream build

**Pull request merged**